lib/gitlab/ci/templates/pipeline/NpmJestCoveragePipeline.yml

```diff
@@ -4,11 +4,14 @@ stages:
   - deploy
 
 build:
-  image: node:16
+  image: $NODE_IMAGE
```

Comment:
It is good in Gitlab templates to add defaults for variables. At the top of this NpmJestCoveragePipeline.yml file can you add the below?

```
variables:
  IMAGE_PREFIX: ""
  NODE_IMAGE: node:16
  DEFAULT_IMAGE: $NODE_IMAGE
  TEST_ARGS: ""
  
default:
  image: ${IMAGE_PREFIX}$DEFAULT_IMAGE
```